docs/en/latest/plugins/serverless.md

```diff
@@ -80,7 +82,7 @@ curl -i http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f03433
     "plugins": {
         "serverless-pre-function": {
             "phase": "rewrite",
-            "functions" : ["return function() ngx.log(ngx.ERR, \"serverless pre function\"); end"]
+            "functions" : ["return function(conf, ctx) ngx.log(ngx.ERR, \"serverless pre function\"); end"]
```

Comment:
I think we should add a new usage example of `conf & ctx` parameter. If we modify the original example, people can easily understand that the `conf & ctx` parameter is necessary.